### PR TITLE
feat(route): digitalcameraworld

### PR DESCRIPTION
**Pull request by EthanWng97**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/digitalcameraworld/news
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/digitalcameraworld/news.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            source: ['digitalcameraworld.com/'],
+        },
+    ],
+    name: 'News',
+    maintainers: ['EthanWng97'],
+    handler,
+};
+
+async function handler() {
+    const rssUrl = `${host}/feeds.xml`;
```
Comment (TonyRL) [inline lib/routes/digitalcameraworld/news.ts:31]: The feed already provides full articles with output much cleaner than this PR.

Comment (EthanWng97) [reply lib/routes/digitalcameraworld/news.ts:31]: seems like newsblur can't parse dc:content

Comment (EthanWng97) [reply lib/routes/digitalcameraworld/news.ts:31]: is there a way to push this changes? seems like some reader can't parse dc:content

Comment (TonyRL) [reply lib/routes/digitalcameraworld/news.ts:31]: You can replace `description` with `dc:content`

Comment (EthanWng97) [reply lib/routes/digitalcameraworld/news.ts:31]: the item.content provided by rss-parser is not correct

Comment (TonyRL) [reply lib/routes/digitalcameraworld/news.ts:31]: Use cheerio with [xmlMode](https://cheerio.js.org/docs/api/interfaces/CheerioOptions).

Comment (EthanWng97) [reply lib/routes/digitalcameraworld/news.ts:30]: done

**Pull request merged**